client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -17,24 +17,31 @@ import Button from 'components/button';
 import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
-import { getJetpackOnboardingProgress, getUnconnectedSiteUrl } from 'state/selectors';
+import Spinner from 'components/spinner';
+import { getJetpackOnboardingProgress, getRequest, getUnconnectedSiteUrl } from 'state/selectors';
 import {
 	JETPACK_ONBOARDING_STEP_TITLES as STEP_TITLES,
 	JETPACK_ONBOARDING_STEPS as STEPS,
 	JETPACK_ONBOARDING_SUMMARY_STEPS as SUMMARY_STEPS,
 } from '../constants';
+import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
 	renderCompleted = () => {
-		const { steps, stepsCompleted } = this.props;
+		const { steps, stepsCompleted, stepsPending } = this.props;
 
 		return map( without( steps, STEPS.SUMMARY ), stepName => {
 			const isCompleted = get( stepsCompleted, stepName ) === true;
+			const isPending = get( stepsPending, stepName );
 			const className = classNames( 'steps__summary-entry', isCompleted ? 'completed' : 'todo' );
 
 			return (
 				<div key={ stepName } className={ className }>
-					<Gridicon icon={ isCompleted ? 'checkmark' : 'cross' } size={ 18 } />
+					{ isPending ? (
+						<Spinner />
```

Comment:
The spinner is `20px` large by default, could we add a `size={ 18 }` to make it as large as the `Gridicon` instances?

Reply:
9267f35